web/scripts/rate_all.py

```diff
@@ -100,8 +99,10 @@ def rate_all(t_from=None, t_to=None, iters=200, lam=.22):
             for k,v in rating_prior.items():
```

Comment:
Is this sensitive to outliers? If you got one unusually bad or good player, would everyone else end up with squished ratings? Maybe it'd be more robust to use the mean and std deviation and rescale that way.